README.md

```diff
@@ -43,6 +43,9 @@ To compile and run the application with an example infrastructure:
 # build and run the entire stack
 docker compose up -d --build
 
+# Add the admin role to the default user
+docker exec osrd-editoast editoast roles add mock/mocked Admin
```

Comment:
1. This won't work if the user did not open the application beforehand. (No request has been sent and so the user does not exist in DB).
2. It's kind of painful to run another command.

An idea could be to adapt the main docker compose to create the user and add it's role before running editoast. Something like:

```yaml
    command:
      - /bin/sh
      - -c
      - "diesel migration run --locked-schema && editoast user add 'mock/mocked' 'Example User' && editoast roles add 'mock/mocked' Admin && exec editoast runserver"
```

We might want to format this command a little bit 😅

Reply:
I implemented your solution in my last commit and it works, thanks!